### PR TITLE
18764-Remove-external-dependencies-from-Kernel-Tests-

### PR DESCRIPTION
**Pull request by demarey**
https://pharo.fogbugz.com/f/cases/18764/Remove-external-dependencies-from-Kernel-Tests

### CODE REVIEW

**Comment by demarey** [comment]
redo stuff done in #365 with current image.
@guillep It looks like Directory class you introduced is not needed anymore (no more dependency on FileSystem in testPrimitiveNamed2. Could you confirm this?
```smalltalk
testPrimitiveNamed2
	"This test useses the #primPathNameDelimiter primitive."

	self compile: '<primitive: ''primitiveDirectoryDelimitor'' module: ''FilePlugin''> ^ #delim' selector: #delim.
	self assert: self delim equals: File delimiter.
```
I did not include Directory class and its test class in this pull request but could be done if needed.

**Comment by Ducasse** [comment]
The error does not seem related. I do not know how to address the error and I think that we should put our heads together and fix it.

File: src/DelaySchedulerDebugUtilities/package.st [hunk 1]
```diff
@@ -0,0 +1 @@
+Package { #name : #DelaySchedulerDebugUtilities }
```
Comment (pavel-krivanek) [inline src/DelaySchedulerDebugUtilities/package.st:1]: The new package that is not mentioned in baselines...

Comment (Ducasse) [reply src/DelaySchedulerDebugUtilities/package.st:1]: Christophe can you check?

**Comment by demarey** [comment]
@pavel-krivanek I added the package to BaselineOfIDE and did a merge to avoid conflicts with current state of pharo development branch

**Comment by pavel-krivanek** [comment]
The CI reports some conflicts

**Comment by pavel-krivanek** [comment]
related failure: DependenciesTests.SystemDependenciesTest

**Comment by pavel-krivanek** [comment]
SystemDependenciesTest>>#testExternalIDEDependencies

File: src/BaselineOfIDE/BaselineOfIDE.class.st [hunk 2]
```diff
@@ -178,6 +178,7 @@ BaselineOfIDE >> baseline: spec [
 		spec package: 'Ring-Deprecated-Tests-Containers'.
 		spec package: 'Ring-Deprecated-Tests-Kernel'.
 		spec package: 'System-DependenciesTests'.
+		spec package: 'DelaySchedulerDebugUtilities'.
```
Comment (pavel-krivanek) [inline src/BaselineOfIDE/BaselineOfIDE.class.st:181]: The package is not mentioned in the baseline groups

**Comment by jecisc** [comment]
Should this be for P7 or P8?

**Comment by guillep** [comment]
I'd move it to Pharo8

**Comment by Ducasse** [comment]
I relaunched the build

**Comment by Ducasse** [comment]
Failing tests are unrelated!

**Comment by Ducasse** [comment]
Code looks simple! Thanks for your patience!

**Comment by estebanlm** [comment]
this was incorrectly merged into Pharo 7.0. I will revert and please can we re-submit this for Pharo 8?

**Pull request merged**